crypto/ml_dsa/ml_dsa_key.c

```diff
@@ -499,6 +499,16 @@ size_t ossl_ml_dsa_key_get_sig_len(const ML_DSA_KEY *key)
     return key->params->sig_len;
 }
 
+OSSL_LIB_CTX *ossl_ml_dsa_key_get0_libctx(const ML_DSA_KEY *key)
+{
+    return key != NULL ? key->libctx : NULL;
+}
+
+void ossl_ml_dsa_key_set0_libctx(ML_DSA_KEY *key, OSSL_LIB_CTX *lib_ctx)
+{
+    key->libctx = lib_ctx;
+}
```

Comment:
This function is not thread safe, and does not have any callers. Let's not add it unless it is actually needed.
```suggestion
```